src/hypothesis/strategies.py

```diff
@@ -205,6 +205,9 @@ def integers(min_value=None, max_value=None):
     from hypothesis.searchstrategy.numbers import IntegersFromStrategy, \
         BoundedIntStrategy, WideRangeIntStrategy
 
+    # Why not a simpler implmentation?  eg:
+    #   `min_int_value = None if min_value is None else ceil(min_value)`
+    # Large values then misbehave under Python 2, so we're inelegant for now
```

Comment:
Cause my fuzz-tests to fail, really.  I've poked at it a bit and decided that it's working and not inelegant enough to make me *want* to trace Python2 int bounding issues.